app/templates/index.html

```diff
@@ -111,7 +111,7 @@
     $('.formatButton').click(function(e){
         e.preventDefault();
         if($(this).hasClass('active')){
-            $(this).removeClass('active')
+           
         } else {
```

Comment:
now that `if` block is unnecessary, it should be removed. you could instead now directly check if it doesn't have the class `active`, and then run the block of code inside `else`